tools/migration/internal/runner_buildonly_test.go

```diff
@@ -34,18 +35,18 @@ func TestMigrationRunner_RunBuildonly(t *testing.T) {
 	})
 
 	t.Run("writes the new version to the new repo, does not install", func(t *testing.T) {
-		dummyLogFile, dummyLogPath := RequireOpenTempFile(t, "logfile")
-		defer RequireRemoveAll(t, dummyLogPath)
+		dummyLogFile, dummyLogPath := repo.RequireOpenTempFile(t, "logfile")
+		defer repo.RequireRemoveAll(t, dummyLogPath)
 		logger := NewLogger(dummyLogFile, false)
-		runner, err := NewMigrationRunner(logger, "buildonly", repoSymlink, "")
+		runner, err := NewMigrationRunner(logger, "buildonly", repoSymLink, "")
 		require.NoError(t, err)
 		runner.MigrationsProvider = testProviderPasses
 
 		runResult := runner.Run()
 		assert.NoError(t, runResult.Err)
 
-		AssertBumpedVersion(t, runResult.NewRepoPath, repoDir, 0)
+		AssertBumpedVersion(t, runResult.NewRepoPath, repoSymLink, 0)
 		assert.Equal(t, uint(1), runResult.NewVersion)
```

Comment:
I didn't understand this the first time around either but didn't follow up enough: why are `"writes the new version to the new repo, does not install"` and `"clones repo"` not combined into a single test?  We run "build-only" in both cases.  Both runs should create the new dir, bump the version, and not link the new dir right?

Reply:
Good catch, done.